xml/admin_ceph_gateway.xml

```diff
@@ -2692,349 +2692,279 @@ rgw_override_bucket_index_max_shards = 12
    </para>
   </sect2>
  </sect1>
- <sect1 xml:id="ceph-rgw-fed">
-<!-- https://github.com/theanalyst/ceph/blob/doc/multisite-rgw/doc/radosgw/multisite.rst -->
 
+ <sect1 xml:id="ceph-rgw-fed">
   <title>Multisite &ogw;s</title>
-
+  <para>&ceph; supports several multi-site configuration options for the
+    &ceph; &ogw;:</para>
   <variablelist>
    <varlistentry>
-    <term>Zone</term>
+    <term>Multi-zone</term>
     <listitem>
      <para>
-      A logical grouping of one or more &ogw; instances. There must be one zone
-      designated as the <emphasis>master</emphasis> zone in a
-      <emphasis>zonegroup</emphasis>, which handles all bucket and user
-      creation.
+      A more advanced configuration consists of one zone group and multiple
```

Comment:
The section _opens_ with "A more advanced configuration consists of..." More advanced than what?